config/schema/elasticsearch_types/uk_market_conformity_assessment_body.json

```diff
@@ -0,0 +1,14 @@
+{
+  "fields": [
+    "uk_market_conformity_assessment_body_name",
+    "updated_at",
+    "uk_market_conformity_assessment_body_number",
+    "uk_market_conformity_assessment_body_type",
+    "uk_market_conformity_assessment_body_registered_office_location",
+    "uk_market_conformity_assessment_body_testing_locations",
+    "uk_market_conformity_assessment_body_website",
+    "uk_market_conformity_assessment_body_email",
+    "uk_market_conformity_assessment_body_phone",
+    "uk_market_conformity_assessment_body_legislative_area"
+  ]
+}
```

Comment:
Do you need to provide any expanded search fields for your drop downs?  (Like in DFID research things: https://github.com/alphagov/rummager/blob/c8bbc6bf2361f37460a9b39b6807e9b7fb9fbaec/config/schema/elasticsearch_types/dfid_research_output.json#L19 ?)

Reply:
good catch, added them in